hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestRpc.java

```diff
@@ -0,0 +1,354 @@
+package org.hypertrace.migration;
+
+import static org.hypertrace.core.span.constants.v1.CensusResponse.CENSUS_RESPONSE_CENSUS_STATUS_CODE;
+import static org.hypertrace.core.span.constants.v1.CensusResponse.CENSUS_RESPONSE_STATUS_CODE;
+import static org.hypertrace.core.span.constants.v1.CensusResponse.CENSUS_RESPONSE_STATUS_MESSAGE;
+import static org.hypertrace.core.span.constants.v1.Envoy.ENVOY_GRPC_STATUS_MESSAGE;
+import static org.hypertrace.core.span.constants.v1.Envoy.ENVOY_REQUEST_SIZE;
+import static org.hypertrace.core.span.constants.v1.Envoy.ENVOY_RESPONSE_SIZE;
+import static org.hypertrace.core.span.constants.v1.Grpc.GRPC_ERROR_MESSAGE;
+import static org.hypertrace.core.span.constants.v1.Grpc.GRPC_REQUEST_BODY;
+import static org.hypertrace.core.span.constants.v1.Grpc.GRPC_RESPONSE_BODY;
+import static org.hypertrace.core.span.constants.v1.Grpc.GRPC_STATUS_CODE;
+import static org.hypertrace.core.span.normalizer.constants.OTelRpcSystem.OTEL_RPC_SYSTEM_GRPC;
+import static org.hypertrace.core.span.normalizer.constants.OTelSpanTag.OTEL_SPAN_TAG_RPC_SYSTEM;
+import static org.hypertrace.core.span.normalizer.constants.RpcSpanTag.RPC_REQUEST_METADATA_AUTHORITY;
+import static org.hypertrace.core.span.normalizer.constants.RpcSpanTag.RPC_REQUEST_METADATA_USER_AGENT;
+import static org.hypertrace.migration.MigrationTestHttp.createSpanFromTags;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.typesafe.config.ConfigFactory;
+import io.jaegertracing.api_v2.JaegerSpanInternalModel.Span;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.stream.Stream;
+import org.hypertrace.core.datamodel.RawSpan;
+import org.hypertrace.core.semantic.convention.constants.error.OTelErrorSemanticConventions;
+import org.hypertrace.core.semantic.convention.constants.rpc.OTelRpcSemanticConventions;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.spannormalizer.jaeger.JaegerSpanNormalizer;
+import org.hypertrace.semantic.convention.utils.rpc.RpcSemanticConventionUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class MigrationTestRpc {
+
+  private final Random random = new Random();
+  private JaegerSpanNormalizer normalizer;
+
+  private Map<String, Object> getCommonConfig() {
+    return Map.of(
```

Comment:
why do we need all kstream related config? I think JeagerSpanNormalizer needs tenantId config.
Shouldn't this suffice?

```
ConfigFactory.parseMap(Map.of("processor", Map.of("defaultTenantId", tenantId)))

```

Reply:
removed